tests/e2e/statefulset_with_topology.go

```diff
@@ -129,4 +129,56 @@ var _ = ginkgo.Describe("[csi-topology-vanilla] Topology-Aware-Provisioning-With
 		fss.DeleteAllStatefulSets(client, namespace)
 	})
 
+	/*
+		Test to verify PV node affinity details such that it should contain both zone and
+		region details of SC.
+		Verify POD is running on the same node as mentioned in node affinity details.
+		Steps
+		1. Create SC with zone and region details specified in the SC
+		2. Create statefulset with replica 3 with the above SC
+		3. Wait for all the stateful set to be up and running
+		4. Wait for pvc to be in bound state and statefulset pod in running state
+		5. Describe pv and verify node affinity details should contain both zone and region details
+		6. Verify POD is running on the same node as mentioned in node affinity details
+		7. Delete statefulset
+		8. Delete PVC and SC
+	*/
+	ginkgo.It("Verify pod and pv topology affinity details", func() {
```

Comment:
Storage policy with single Zone and region details in the allowed topology

Reply:
done